addon/helpers/without.js

```diff
@@ -16,7 +15,7 @@ export function without(needle, haystack) {
     return false;
   }
 
-  if (typeOf(needle) === 'array' && get(needle, 'length')) {
+  if (isEmberArray(needle) && get(needle, 'length')) {
```

Comment:
We could get around that with an or check
`if ((isEmberArray(needle) || typeOf(needle) === 'array') && get(needle, 'length'))`